--- a/configure	2025-04-20 14:03:23.072889579 -0700
+++ b/configure	2025-04-20 14:04:04.540844142 -0700
@@ -12981,7 +12981,7 @@
   hardcode_into_libs=yes
 
   # Add ABI-specific directories to the system library path.
-  sys_lib_dlsearch_path_spec="/lib64 /usr/lib64 /lib /usr/lib"
+  sys_lib_dlsearch_path_spec="/lib /usr/lib"
 
   # Ideally, we could use ldconfig to report *all* directores which are
   # searched for libraries, however this is still not possible.  Aside from not
@@ -16973,7 +16973,7 @@
   hardcode_into_libs=yes
 
   # Add ABI-specific directories to the system library path.
-  sys_lib_dlsearch_path_spec="/lib64 /usr/lib64 /lib /usr/lib"
+  sys_lib_dlsearch_path_spec="/lib /usr/lib"
 
   # Ideally, we could use ldconfig to report *all* directores which are
   # searched for libraries, however this is still not possible.  Aside from not
